docker/starlight.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Node for Tanssi
+#
+# Requires to run from repository root and to copy the binary in the build folder (part of the release workflow)
+
+FROM docker.io/library/ubuntu:20.04 AS builder
+
+RUN apt-get update && apt-get install -y ca-certificates lsof && update-ca-certificates
+
+FROM debian:bookworm-slim
+LABEL maintainer "gorka@moondancelabs.com"
+LABEL description="Binary for Dancelight"
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /tanssi-relay tanssi-relay && \
+	mkdir -p /tanssi-relay/.local/share && \
+	mkdir /data && \
+	chown -R tanssi-relay:tanssi-relay /data && \
+	ln -s /data /tanssi-relay/.local/share/tanssi-relay && \
+	rm -rf /usr/sbin
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+USER tanssi-relay
+
+COPY --chown=tanssi-relay build/tanssi-relay* /tanssi-relay
+RUN chmod uog+x /tanssi-relay/tanssi-relay*
+
+# 30333 for parachain p2p
```

Comment:
This port is not needed, also these are the defaults of `tanssi-node`, they may be different in `tanssi-relay`

Reply:
I think by default we need to expose: 30333 9933 9944 9615. Based of https://github.com/paritytech/polkadot-sdk/blob/master/substrate/scripts/ci/docker/substrate.Dockerfile

Reply:
AND https://github.com/paritytech/polkadot-sdk/blob/master/docker/dockerfiles/polkadot/polkadot_builder.Dockerfile

Reply:
Updated in https://github.com/moondance-labs/tanssi/pull/698/commits/648e4b9cad9922a8972c3cc81740afae6764b732